### PR TITLE
add keycodes 21 25 47 95 168 172

### PR DESCRIPTION
**Pull request by kevindavus**
addresses:
https://github.com/wesbos/keycodes/issues/149
https://github.com/wesbos/keycodes/issues/142
https://github.com/wesbos/keycodes/issues/141
https://github.com/wesbos/keycodes/issues/140
https://github.com/wesbos/keycodes/issues/125
https://github.com/wesbos/keycodes/issues/124
https://github.com/wesbos/keycodes/issues/100
https://github.com/wesbos/keycodes/issues/99
https://github.com/wesbos/keycodes/issues/98
https://github.com/wesbos/keycodes/issues/80  (American/Polish but not Norweigan)
https://github.com/wesbos/keycodes/issues/74

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ var keyCodes = {
   18 : "alt",
   19 : "pause/break",
   20 : "caps lock",
+  21: "hangul",
```
Comment (anselmh) [inline scripts.js]: You might want to stick to the code convention used here, note the space before the `:`.

File: scripts.js [hunk 2]
```diff
@@ -72,6 +75,7 @@ var keyCodes = {
   91 : "Windows Key / Left ⌘ / Chromebook Search key",
   92 : "right window key ",
   93 : "Windows Menu / Right ⌘",
+  95: "sleep ",
```
Comment (anselmh) [inline scripts.js]: Why the empty space in the value? Seems unnecessary.

Comment (osfan501) [inline scripts.js]: I looked at the wrong line when deciding on the space before quote. I fixed a few of those

Comment (anselmh) [inline scripts.js]: ❤️

**Comment by wesbos** [comment]
thanks!

**Pull request merged**